### PR TITLE
Save hveto round signficance values in a txt file

### PR DESCRIPTION
**Pull request by siddharth101**
For round 2 onwards,  saving the old and new significance values in a txt file. 
An example run can be found [here](https://ldas-jobs.ligo-la.caltech.edu/~siddharth.soni/hvetooutput/hvetotestdec32019/significance/)

### CODE REVIEW

File: hveto/__main__.py [hunk 1]
```diff
@@ -550,6 +552,28 @@ def main(args=None):
         #   only now do we actually have the new data to
         #   calculate significance drop
         if rnd.n > 1:
+            oldsigfile = os.path.join(
+                signidir,
+                '%s-HVETO_OLD_SIGNIFICANCE_TRIGS_ROUND_%d-%d-%d.txt' % (
+                    ifo, rnd.n, start, duration))
+            newsigfile = os.path.join(
+                 signidir,
+                 '%s-HVETO_NEW_SIGNIFICANCE_TRIGS_ROUND_%d-%d-%d.txt' % (
```
Comment (duncanmmacleod) [inline hveto/__main__.py:561]: Here `HVETO_{NEW,OLD}_SIGNIFANCE_TRIGS` makes more sense (to me) as `HVETO_SIGNIFICANT_CHANNELS` (these aren't really triggers.

Comment (siddharth101) [reply hveto/__main__.py]: @duncanmmacleod Thanks for these suggested changes, the latest commit incorporates them. [Here's](https://ldas-jobs.ligo-la.caltech.edu/~siddharth.soni/hvetooutput/hvetotestfeb_20_2020/significance/) an example run based on this.

**Comment by jrsmith02** [comment]
I was wondering if the rounding to two decimal points is too aggressive. For example, round(0.000001, 2) gives 0.0 so if there is a channel with very low significance less than 0.005 it will show as 0.0 in the files.  That might mislead people into thinking there are no coincidences. But maybe I’m overthinking it.

**Comment by siddharth101** [comment]
Thats a fair thought @jrsmith02 . I reran the Feb20 analysis [here](https://ldas-jobs.ligo-la.caltech.edu/~siddharth.soni/hvetooutput/hvetotestfeb_20a_2020/significance/) with rounding upto 5 digits. For any round there are no values below 0.01, the minimum significance across all rounds is 0.089650. How about we round up to 4 digits, and then leave further rounding to anyone analyzing the data.

**Comment by jrsmith02** [comment]
That's ok with me. Thanks.

**Pull request merged**